test/specs/spec.js

```diff
@@ -0,0 +1,26 @@
+const expect = require("chai").expect;
+const PageFactory = require("../utils/pageFactory");
+const EC = protractor.ExpectedConditions;
+describe("Heroes home page tests", function () {
+
+    beforeEach(function() {
+            browser.ignoreSynchronization = true;
+            return browser.manage().window().maximize();
+    });
+
+    it("should have 11 recent badges", async function () {
+        await PageFactory.getPage("Login").open();
```

Comment:
Please move pages declaration to separate constants:
```
const loginPage = Pagefactory.getPage("Login");
// some code
await loginPage.open();
await logiPAge.typrLogin();

Reply:
done